_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -53465,7 +53465,8 @@
 /turf/open/floor/plasteel,
 /area/hallway/secondary/exit/departure_lounge)
 "dUk" = (
-/obj/machinery/cryopod{
+/obj/machinery/cryopod/perma{
+	icon_state = "cryopod-open";
```

Comment:
dirty map var edit. Remove it.

Reply:
I'll try again tonight, but I did not var edit anything on the maps, what you see is the result of placing it down and nothing else.